package.json

```diff
@@ -26,9 +31,8 @@
     "watch": "node --test --watch"
   },
   "dependencies": {
-    "stylelint-config-recommended-scss": "^16.0.1",
-    "stylelint-config-standard": "^39.0.0",
-    "stylelint-scss": "^6.13.0"
```

Comment:
I guess you removed `stylelint-scss` because it is already a dependency of `stylelint-config-recommended-scss`?

I hope that removing it does not cause any problems.

Reply:
It was added in https://github.com/stylelint-scss/stylelint-config-standard-scss/pull/260 and isn't needed once things are synced up.

Reply:
alright, then everything is good 👍